pages/validators/index.vue

```diff
@@ -26,7 +26,7 @@
       :delegations="delegations"
       :rewards="rewards"
       :search-term="searchTerm ? true : false"
-      :loaded="delegationsLoaded"
+      :loaded="validatorsLoaded"
```

Comment:
```suggestion
      :loaded="validatorsLoaded && delegationsLoaded"
```

Reply:
is delegationsLoaded not set?

Reply:
why should validator table block on loading delegations?